src/main/java/org/example/DBController.java

```diff
@@ -128,6 +129,142 @@ public boolean updateCustomerFunds(int customerId, int newAmount) {  //void
         return false;
     }
 
+    public void createTransactionAudit(int senderId, String senderName, int moneyInCents, int getterId, String getterName) throws SQLException {
+        String sql = "INSERT INTO transactions_audit (sender_id, sender_nickname, amount_sent, receiver_id, receiver_nickname) " + "VALUES (?, ?, ?, ?, ?)";
+
+        PreparedStatement ps = connection.prepareStatement(sql,
+                Statement.RETURN_GENERATED_KEYS);
+        {
+
+            ps.setInt(1, senderId);
+            ps.setString(2, senderName);
+            ps.setInt(3, moneyInCents);
+            ps.setInt(4, getterId);
+            ps.setString(5, getterName);
+
+            int rows = ps.executeUpdate();                          // returns 1
+            System.out.println("Rows inserted: " + rows);

```

Comment:
I would rename the variable to `rowsInserted` just so it is a little clearer. `rows` implies an entire row data structure with columns. `rowsInserted` implies just a number.
```suggestion
            System.out.println("Rows inserted: " + rowsInserted);

```